src/main/kotlin/com/example/Food_API/controller/FoodController.kt

```diff
@@ -7,18 +7,19 @@ import org.springframework.web.bind.annotation.*
 @RestController
 @RequestMapping("/api/food")
 class FoodController() {
-    @GetMapping(produces = ["application/json"])
-    fun getRecipes(): String? {
+
+    @GetMapping("/{foodId}")
+    fun findFoodById(@PathVariable foodId: Int): String? {
```

Comment:
I would suggest moving the logic for this method into some sort of service interface, maybe a `FoodService` interface? That way if you do add more functions to this controller (which I'm sure you will) the controller will be easier to read and more cleaned up.

Reply:
sounds good! That was going to be my next step was to add the logic to the service interface.

Reply:
👍